components/sst_importer/src/sst_writer.rs

```diff
@@ -197,20 +235,27 @@ impl<E: KvEngine> RawSstWriter<E> {
                         return Err(crate::Error::TtlNotEnabled);
                     };
 
-                    for m in batch.take_pairs().into_iter() {
+                    for mut m in batch.take_pairs().into_iter() {
+                        self.check_api_version::<API>(m.get_key())?;
+                        let key = API::encode_raw_key_owned(
+                            m.take_key(),
+                            Some(TimeStamp::new(batch.get_ts())),
```

Comment:
if user didn't set ts field. will it be zero?
Maybe we should set it to none?

Reply:
There is no difference, because with `ApiVersion` of `ApiV1` or `ApiV1Ttl`. The encoding will ignore the `ts`, otherwise, with ApiV2 the `ts` will actually make sense.